sql/blocks/blocks.go

```diff
@@ -71,6 +71,25 @@ func Get(db sql.Executor, id types.BlockID) (rst *types.Block, err error) {
 	return rst, err
 }
 
+func LastValid(db sql.Executor) (types.LayerID, error) {
+	var lid types.LayerID
+	// it doesn't use max(layer) in order to get rows == 0 when there are not layers
```

Comment:
I don't understand the subtle difference here, could you explain?

Btw a typo:
```suggestion
	// it doesn't use max(layer) in order to get rows == 0 when there are no layers
```

Reply:
max(layer) aggregation will always return rows = 1, even if nothing exists. i will add to the comment